src/test/scala/org/broadinstitute/dsde/firecloud/mock/MockGoogleServicesDAO.scala

```diff
@@ -1,20 +1,15 @@
 package org.broadinstitute.dsde.firecloud.mock
 
-import java.io.{ByteArrayInputStream, InputStream}
-import java.util.concurrent.LinkedBlockingQueue
-
-import akka.actor.ActorRefFactory
-import akka.http.scaladsl.model.{HttpResponse, StatusCodes}
-import org.broadinstitute.dsde.firecloud.FireCloudException
+import akka.http.scaladsl.model.HttpResponse
```

Comment:
is there a reason these imports are added even though no code in this file has changed?

Reply:
ah - after removing the methods, I used IntelliJ's "optimize imports" to clean it up. This `HttpResponse` import is just moved down from the previous commit's line 7.

Reply:
aha I see it rearranged them in such a way that I even thought I did a search and find, I didnt see them :D